frontend/src/pages/Overview.js

```diff
@@ -47,7 +49,16 @@ export default function Overview({ jwtToken }) {
             startDate: '31.01.2020',
             winner: 'Anna',
         },
-    ];
+    ];*/
+
+    const [sessions, setSessions] = useState([]);
+
+    useEffect(() => {
+        gameNightTrackerApi
```

Comment:
When `jwtToken` is undefined this will send an api request that is guaranteed to fail, maybe check if it is defined before hitting your api?